js/main.js

```diff
@@ -0,0 +1,48 @@
+const addBooks = document.querySelector('#add');
+const bookName = document.getElementById('title');
+const authorName = document.getElementById('author');
+
+let bookList = [];
```

Comment:
- [ ] For us not to keep reassigning this variable and making reference to a null value, refactor as below.
```suggestion
let bookList = JSON.parse(localStorage.getItem('books')) || [];
```

Reply:
This issue is well resolved! ✔️